packages/shared/src/utils/url/helpers.js

```diff
@@ -15,6 +15,8 @@ export const getUrlSmartTrader = () => {
     return `${base_link}/${i18n_language.toLowerCase()}/trading.html`;
 };
 
+export const getUrlBinaryBot = () => (isStaging() ? deriv_urls.BINARYBOT_STAGING : deriv_urls.BINARYBOT_PRODUCTION);
```

Comment:
```suggestion
export const getUrlBinaryBot = () => (isProduction() ? deriv_urls.BINARYBOT_PRODUCTION : deriv_urls.BINARYBOT_STAGING);
```

To make sure production target is the one that only runs for production, for testlinks, local environment, etc. Is it possible to use staging?